src/web/lib/ArticleRenderer.tsx

```diff
@@ -194,23 +194,45 @@ export const ArticleRenderer: React.FC<{
 				case 'model.dotcomrendering.pageElements.EmbedBlockElement':
 					if (!element.safe) {
 						return (
-							<Figure role={element.role}>
-								<UnsafeEmbedBlockComponent
-									key={i}
-									html={element.html}
-									alt={element.alt || ''}
-									index={i}
-								/>
+							<Figure
+								role={element.role}
+								id={`embed-block-element-${i}`}
+							>
+								<ClickToView
+									role={element.role}
+									isTracking={element.isThirdPartyTracking}
+									source={element.source}
+									sourceDomain={element.sourceDomain}
+									abTests={abTests}
+								>
+									<UnsafeEmbedBlockComponent
+										key={i}
+										html={element.html}
+										alt={element.alt || ''}
+										index={i}
+									/>
+								</ClickToView>
 							</Figure>
 						);
 					}
 					return (
-						<Figure role={element.role}>
-							<EmbedBlockComponent
-								key={i}
-								html={element.html}
-								alt={element.alt}
-							/>
+						<Figure
+							role={element.role}
+							id={`embed-block-element-${i}`}
+						>
+							<ClickToView
+								role={element.role}
+								isTracking={element.isThirdPartyTracking}
+								source={element.source}
+								sourceDomain={element.sourceDomain}
+								abTests={abTests}
+							>
+								<EmbedBlockComponent
+									key={i}
```

Comment:
I think this key can move up now to the `Figure`? This was quite possibly already an issue prior to this PR to be fair!